Makefile

```diff
@@ -188,10 +188,11 @@ dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-build-epoch.txt:
 	echo $(SOURCE_DATE_EPOCH) > dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-build-epoch.txt
 
 # Build the HTML and Markdown documentation from the package's source.
+DOCS_SOURCE := $(shell find docs/source -type f -name '*.rst')
 .PHONY: docs docs-html docs-md
 docs: docs-html docs-md
 docs-html: docs/_build/html/index.html
-docs/_build/html/index.html: check test
```

Comment:
Add the dependencies to the Markdown docs too, methinks.

Reply:
See commit [0b48d87](https://github.com/jenstroeger/python-package-template/pull/560/commits/0b48d874ecb1d95e2024d650176f43314357b969).